src/main/kotlin/gg/skytils/skytilsmod/listeners/DungeonListener.kt

```diff
@@ -176,6 +179,10 @@ object DungeonListener {
                         val partyCount = partyCountPattern.find(tabEntries[0].second)?.groupValues?.get(1)?.toIntOrNull()
                         if (partyCount != null) {
                             println("There are $partyCount members in this party")
+                            if (team.size != partyCount) {
+                                println("Clearing team as the party size has changed ${team.size} -> $partyCount")
+                                team.clear()
+                            }
```

Comment:
Why is all this changed? It seems unrelated

Reply:
this is caused by me not merging the latest commit from the branch I think xd